### PR TITLE
Initial Commit for CMS Program Tests

### PR DESCRIPTION
**Pull request by dczulada**
Pull requests into Cypress require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [ ] This pull request describes why these changes were made.
- [ ] Internal ticket for this PR:
- [ ] Internal ticket links to this PR
- [ ] Code diff has been done and been reviewed
- [ ] Tests are included and test edge cases
- [ ] Tests have been run locally and pass

**Reviewer 1:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

**Reviewer 2:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

### CODE REVIEW

File: config/cypress.yml [hunk 1]
```diff
@@ -141,6 +142,21 @@ certifications:
...
       - 'FilteringTest'
 tests:
+  CMSProgramTest:
+    title: 'CMS Implementation Guide Checklist'
+    instructions:
+      - instruction: 'Enter the information below into your EHR and record the entered values in the inputs boxes provided.'
+        subinstructions:
+          - 'The "CMS IG Conformance Statement Reference" can be used to find specific guidance in the CMS Implementation Guide.'
+          - 'This information may need to be entered into a patient record, a provider profile or as a system configuration.'
+      - instruction: 'Export (at least) one QRDA file that contains the information below and upload to Cypress.'
```
Comment (lmd59) [inline config/cypress.yml]: instructions should indicate that files should be put in a .zip
